src/storages/types.ts

```diff
@@ -355,6 +360,15 @@ export interface IImpressionCountsCacheSync extends IRecorderCacheProducerSync<R
   pop(toMerge?: Record<string, number> ): Record<string, number> // pop cache data
 }
 
+export interface IImpressionCountsCacheInRedis extends IImpressionCountsCacheSync {
+  postImpressionCountsInRedis(body: string): Promise<[Error | null, any][]>
+}
+
+export interface IImpressionCountsCacheAsync extends IImpressionCountsCacheBase, IRecorderCacheProducerAsync<Record<string, number>> {
```

Comment:
Remove this interface. Lets use `IImpressionCountsCacheSync` for now, because ImpressionCountInRedis extends the one in memory